app/src/main/java/org/systers/mentorship/view/activities/MainActivity.kt

```diff
@@ -96,11 +98,33 @@ class MainActivity: BaseActivity() {
         outState.putBoolean("atHome", atHome)
     }
 
+    private var doubleBackToExitPressedOnce = false
```

Comment:
Is a global variable here the best way to keep track of this change? We can probably model the double backpress as an Rx stream using the RxBinding library, I guess. 
cc @m-murad @isabelcosta Your input on this would be greatly appreciated.

Reply:
Besides, maybe we can find a more descriptive name for this variable if we do have to use a global variable, after all.